static/js/settings_bots.js

```diff
@@ -80,14 +69,6 @@ function add_bot_row(info) {
     }
 }
 
-function is_local_part(value, element) {
-    // Adapted from Django's EmailValidator
-    return (
-        this.optional(element) ||
-        /^[\w!#$%&'*+/=?^`{|}~-]+(\.[\w!#$%&'*+/=?^`{|}~-]+)*$/i.test(value)
-    );
-}
```

Comment:
We are not checking email characters currently. For example having "@" in email used to raise error previously, but it succeeds currently.

Reply:
Should we handle this at the backend?

Reply:
At backend we only check string empty or not.

Reply:
that deleted regex allows special characters, it does not allow "@" but it allows "#", "[" etc, is it okay for bot email, as on form has mentioned that (a-z, 0-9, and dashes only), should I modify that regex or just use the same?